### PR TITLE
MB-11031 - Remove PPM Test from Cypress

### PR DESCRIPTION
**Pull request by pearl-truss**
…for PPMs

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the
2. Login as a
3.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

**Review by felipe-lee — COMMENTED** [review]
Awesome, thanks for cleaning these up. I see a few other places PPMs are referenced like in 

* https://github.com/transcom/mymove/blob/pm-mb-11031-delete-ppm-cypress-code/cypress/support/commands.js
* https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/mymove/landingPages.js#L18-L59
* https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/mymove/homepage.js#L21-L27
* https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/office/cancelAndStartNewMove.js#L7

**Comment by pearl-truss** [comment]
> Awesome, thanks for cleaning these up. I see a few other places PPMs are referenced like in
> 
> * https://github.com/transcom/mymove/blob/pm-mb-11031-delete-ppm-cypress-code/cypress/support/commands.js
> * https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/mymove/landingPages.js#L18-L59
> * https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/mymove/homepage.js#L21-L27
> * https://github.com/transcom/mymove/blob/c159370dfbbeff37d8897dce1553278f612500ca/cypress/integration/office/cancelAndStartNewMove.js#L7

Thanks for reviewing, this has been addressed

**Comment by felipe-lee** [comment]
@pearl-truss , I think there's one more:

**Comment by pearl-truss** [comment]
> apiSignInAsUser

Is already defined on line 126. Can you elaborate on the change that you're expecting? Because I might have gone down the wrong path and haven't been able to get the integration test to pass @felipe-lee

**Comment by felipe-lee** [comment]
> > apiSignInAsUser
> 
> Is already defined on line 126. Can you elaborate on the change that you're expecting? Because I might have gone down the wrong path and haven't been able to get the integration test to pass @felipe-lee

Sorry, I meant since the existing function uses a ppm route, we should stop using it, but there are a few places that seem to need some sign in functionality so I was thinking it might be possible to replace the usages of `apiSignInAsPpmUser` with the other existing sign in function `apiSignInAsUser`, but I haven't tried it so it might not actually work well as a direct replacement.

**Review by felipe-lee — APPROVED** [review]
Awesome, thanks for cleaning up these files! Now to wait for circleci to start working again...

File: cypress/integration/mymove/orders.js [hunk 1]
```diff
@@ -58,31 +58,31 @@ describe('orders entry', function () {
     });
   });
 
-  it('will allow amended orders upload', function () {
-    const userId = '6016e423-f8d5-44ca-98a8-af03c8445c94';
-    cy.apiSignInAsUser(userId);
-    cy.intercept('PATCH', '**/internal/orders/**').as('uploadAmendedOrder');
-    cy.intercept('GET', '**/internal/service_members/**').as('currentOrders');
+  // it('will allow amended orders upload', function () {
```
Comment (felipe-lee) [inline cypress/integration/mymove/orders.js:61]: Is it ok to not have an e2e test for amended orders? tbh, I don't remember if this is a thing that works right now or not. Like I know there are bugs around updating some orders info, but I thought uploading amended orders worked. I could definitely be wrong though.

**Comment by felipe-lee** [comment]
Thanks for updating that, if all passes, I'm good with this merging!

**Comment by felipe-lee** [comment]
I think now that https://github.com/transcom/mymove/pull/8095 merged, you can pull in master again to fix the issue.

**Pull request merged**